lib/util.js

```diff
@@ -207,11 +207,11 @@ function formatWithOptions(inspectOptions, f) {
             break;
           case 111: // 'o'
           {
-            const opts = Object.assign({}, inspectOptions, {
+            const opts = { ...inspectOptions, ...{
```

Comment:
Spreading an object literal is unnecessary. Just add the properties to the top level object literal

Reply:
Good catch. Fixed.